coderd/coderd.go

```diff
@@ -405,15 +408,16 @@ func New(options *Options) *API {
 		r.Post("/csp/reports", api.logReportCSPViolations)
 
 		r.Get("/buildinfo", buildInfo)
+		r.Route("/deployment", func(r chi.Router) {
+			r.Use(apiKeyMiddleware)
+			r.Get("/config", api.deploymentConfig)
```

Comment:
Breaking change? `/config/deployment` => `/deployment/config`.

Reply:
Since this endpoint is only relied on in the dashboard, I wouldn't consider this breaking, but if you think it is I'm fine to put `!` with it!

Reply:
No strong preference, just being cautious. 😄